g11n-ws/modules/md-service-authen/src/main/java/com/vmware/vip/core/validation/ParameterValidation.java

```diff
@@ -17,6 +21,13 @@
 
 public class ParameterValidation implements IVlidation {
 	HttpServletRequest request = null;
+	public final static String TAG_BUNDLE_BASE_PATH = "bundleBasedPath";
+	public final static String BUNDLE_FILE = "bundle.json";
```

Comment:
Looks like this PR doesn't include code for generating this file.

Reply:
it's done by build.gradle in translation.git project.